doc/ref_model/chapters/chapter08.md

```diff
@@ -790,6 +790,33 @@ The following five test categories have been identified as minimal testing requi
 
 <p>Respective test cases can be found in the <a title="Annex for Chapter 8" href="https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter08-annex.md" target="_blank" rel="noopener">Annex for Chapter 8</a>.</p>
 
+
+### 8.7.10 Test Case Selection Requirements
```

Comment:
Add an anchor tag on heading as done for others

Reply:
thanks, will add this.